tests/tests_pytorch/strategies/test_sharded_strategy.py

```diff
@@ -321,3 +354,33 @@ def test_ddp_sharded_strategy_checkpoint_multi_gpu_fairscale_optimizer(tmpdir, s
     # Assert model parameters are identical after loading
     for trained_param, loaded_param in zip(model.parameters(), saved_model.parameters()):
         assert torch.equal(trained_param.to("cpu"), loaded_param)
+
+
+@RunIf(min_cuda_gpus=2, fairscale=True)
+def test_ddp_sharded_strategy_fit_ckpt_path_downsize_gpus(tmpdir):
```

Comment:
Note: This test takes ~17 seconds to finish.

Reply:
oh! that's too much. wondering if we should create another one `slow-gpu` so these could run separately.

Reply:
cc: @carmocca @Borda

Reply:
The current workflow could be ported to run on Azure with GPUs. This wasnt done in the past because the GPU bill was expensive, but now that it runs on our cluster it should be okay. I suggest you open an issue

Reply:
done #15372